lib/turn.rb

```diff
@@ -0,0 +1,32 @@
+class Turn
+    attr_reader :guess
+    
+    def initialize(string, card)
+        @card = card
+        @guess = string
+    end
+
+    def card()
```

Comment:
If your method doesn't take a parameter/argument, you don't need empty parentheses here. You can just write the following instead:
```suggestion
    def card
      @card 
     end 
```